tasks/main.yml

```
@@ -5,3 +5,11 @@
     ansible_python_interpreter: "{{ os_networks_venv }}/bin/python"
   environment:
     OS_IDENTITY_API_VERSION: 3
+
+- name: Import rbac.yml
+  ansible.builtin.import_tasks: rbac.yml
```

Use a dynamic include to avoid all the skipped tasks
```suggestion
  ansible.builtin.include_tasks: rbac.yml
```